ci-operator/config/openshift/cluster-capi-operator/openshift-cluster-capi-operator-main.yaml

```
@@ -71,43 +71,67 @@ tests:
 - as: e2e-aws-ovn-techpreview
   steps:
     cluster_profile: aws
-    workflow: openshift-e2e-aws-capi
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
+    workflow: openshift-e2e-vsphere-capi
```

Needs to stay as the correct platform here
```suggestion
    workflow: openshift-e2e-aws
```